tests/robot-tests/tests/visual_testing/tables_and_charts.robot

```diff
@@ -0,0 +1,163 @@
+*** Settings ***
+Resource    ../libs/common.robot
+Resource    ../libs/charts.robot
+Library     ../libs/visual.py
+Library     tables_and_charts.py
+
+*** Variables ***
+${SNAPSHOT_FOLDER}=             snapshots/%{RUN_IDENTIFIER}
+${KEY_STATS_FOLDER}=            key-stats
+${SECONDARY_STATS_FOLDER}=      secondary-stats
+${CONTENT_SECTIONS_FOLDER}=     content-sections
+${FAST_TRACKS_FOLDER}=          fast-tracks
+
+*** Keywords ***
+Check release
+    [Arguments]    ${release}
+
+    Log to console    \n\n\t=====================================================================\n
+    Log to console    \tProcessing release at %{PUBLIC_URL}${release.url}
+
+    user navigates to public frontend    %{PUBLIC_URL}${release.url}
+
+    IF    ${release.has_key_stat_blocks} is ${TRUE}
+
+        Log to console    \n\tCapturing Key Stats:
+
+        user waits until page contains element    xpath://h2[contains(text(), "Headline facts and figures")]
+        user scrolls to element    xpath://h2[contains(text(), "Headline facts and figures")]
+        user waits until page contains element    id:releaseHeadlines-summary
+
+        FOR    ${content_block}    IN    @{release.key_stat_blocks}
+            check key stats table    ${content_block}
+        END
+    END
+
+    IF    ${release.has_secondary_stat_blocks} is ${TRUE}
+        Log to console    \n\tCapturing Secondary Stats:
+
+        FOR    ${content_block}    IN    @{release.secondary_stat_blocks}
```

Comment:
Just wondering if the `@` in `@{release.secondary_stat_blocks}` is a typo?

Reply:
No no, it's using `release.secondary_stat_blocks` as a loop variable, therefore needing the `@` in front.